docker-compose/keycloak.yaml

```diff
@@ -0,0 +1,39 @@
+services:
+  keycloak:
+    image: quay.io/keycloak/keycloak:22.0.5
+    command:
+      - "start-dev"
+    environment:
+      KEYCLOAK_ADMIN: admin
+      KEYCLOAK_ADMIN_PASSWORD: admin
+      KC_HEALTH_ENABLED: true
+      KC_HOSTNAME_STRICT_BACKCHANNEL: false
+      KC_HOSTNAME: localhost
+    ports:
+      - 1852:8080
+    healthcheck:
+      test: ["CMD-SHELL", "exec 3<>/dev/tcp/127.0.0.1/8080;echo -e \"GET /health/ready HTTP/1.1\r\nhost: http://localhost\r\nConnection: close\r\n\r\n\" >&3;grep \"HTTP/1.1 200 OK\" <&3"]
+      interval: 3s
+      timeout: 1s
+      retries: 30
+
+  keycloak-setup:
+    image: quay.io/adorsys/keycloak-config-cli:5.9.0-22.0.4
+    user: root
+    entrypoint: ["/bin/bash", "-c", "mkdir -p /config && /tmp/keycloak_custom_role_creation.sh /tmp/local-development.json /config/local-development.json && exec java $JAVA_OPTS -jar /app/keycloak-config-cli.jar $0 $@"]
+    depends_on:
+      keycloak:
+        condition: service_healthy
+    volumes:
+      - ./config/keycloak:/tmp
+    environment:
+      - KEYCLOAK_URL=http://keycloak:8080
```

Comment:
sometimes we are using localhost and sometimes keycloak. We are sure its not a problem?

Reply:
There were some issues with mixed ports on localhost/in docker. So I changed the internal port to 1852 as well.
But it seems to work as intended with the mix of localhost and keycloak, both when running all of the infrastructure, and a flink-job inside and outside docker.